src/scripting/run_script.cc

```diff
@@ -80,7 +82,9 @@ run_string_as_script(lua_State* L, const std::string& identifier, const std::str
 	lua_setglobal(L, "__file__");
 
 	std::unique_ptr<LuaTable> return_value(new LuaTable(L));
-	lua_pop(L, 1);
+	if (!no_pop_table) {
+		lua_pop(L, 1);
```

Comment:
Is lua_pop() necessary at all? Or could it just be deleted (instead of conditionally be skipped)? After pop, the "import()" function is topmost on the stack, and I doubt any code was relying on this.

Reply:
It is necessary ... (see https://github.com/widelands/widelands/pull/6210#issuecomment-1871264388)